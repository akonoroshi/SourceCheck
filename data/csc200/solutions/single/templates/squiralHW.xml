<project name="squiralHW" app="iSnap 2.1, http://go.ncsu.edu/isnap" version="1" guid="28495c21-e933-460a-a91e-dd9348770d8c" assignment="polygonMakerLab"><data>{}</data><notes></notes><thumbnail>data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAKAAAAB4CAYAAAB1ovlvAAAF3klEQVR4Xu2dPUhkZxSGzxiVYEYRWQhiyigkGAvxFwtBrQIptFDTKEFQsLMVJkWUmM6AgsIWgkVQtwlpUqnTKGpAg2hAWZR0iiApNiJaTLgXdnZ+NPde7/ycn3eaRfzud99znsfz7XWEiSQSiQThhQ4UqQMRCFikzuO2bgcgIEQoageeFPDo6Iji8Xheg3V2dlJHR0de74HN+XcgS8CNjQ1qbm4OnPztm5/onz9+pYjHlfVffEVV3712V93c3JAjIl52O5Am4N7eHlVVVZHzXBKJRAL9+/hzL31eU+6rk3+P/J7cf2dnh8bHx31dh0X6OpAmYCwWo4GBAVcO5xh2jsixsTH3693dXVpZWaGDgwNaWlqi1dVVGhkZocPDQ/f7la+/pk+jpckOfTJ9QJGPo/Tu+y+zuvb229+Scu/v79PExIS+zqIiXx1IE3B+fp66urpcOY6Pj+ni4oLm5uaop6eH6uvr6fz8nBoaGqiiooJaW1upqamJ7u7u3PWvVvvTBIz+8Bcl7t/Rvz+2ZQU5HXiTnICbm5s0PT3tKywW6etAmoC3t7d0enr67PF7eXlJo6OjdHZ2RtfX12nrPlsfolcVH/nq0J/f/JKcgOXl5dTWli2pr42wSHwHsh5ClpeX3WkX9NXe3k7RaNTzsvv7e3L+3+e8tre3aXZ21vMaLNDbAfweUC9bEZVBQBGY9IaEgHrZiqgMAorApDckBNTLVkRlEFAEJr0hIaBetiIqg4AiMOkNCQH1shVRGQQUgUlvSAiol62IyiCgCEx6Q0JAvWxFVAYBRWDSGxIC6mUrojIIKAKT3pAQUC9bEZVBQBGY9IaEgHrZiqgMAorApDckBNTLVkRlEFAEJr0hIaBetiIqg4AiMOkNCQH1shVRGQQUgUlvSAiol62IyiCgCEx6Q0JAvWxFVAYBRWDSGxIC6mUrojIIKAKT3pAQUC9bEZVBQBGY9IaEgHrZiqgMAorApDekKQGdT4Da2tqi6upqvUSFVWZKQIdNf3+/i6ikpIRKS0tpfX1dGDJdcc0J6EzAhYWFLIqPj4/U29tLU1NTuggzr8acgM4nMy0uLmZheXh4oL6+PhocHKS6ujrm2PTEMydg6hFcVlZGa2traTSdI3loaEgPYeaVmBKwpaXF/XSmysrKZ7FAwMIaa0pAP63NFDBzQvrZI8wa5/P6Jicnw2wh6loImIGr2BPQEX54eFiURGHCQkAIGMaf0NdCwAACOh/UXVNTE7rpmRukPvRgAua8vbI2/L8juBByFOIenIhgAgaYgIWQoxD3gICcOgABi0oDE/CFAoZ5Wi72MV9U4zJuDgEhYFF9hIAQEAIWtQM5EtDrHZPUdzhwBH9oOiZgjgT0+iFKfbqFgBDwWV/8yhH0IQQCPt1yTMA8TMD3f2X93DscfiX3mqoavg8B8yDgU2JgAmIC+hoYfqcTjmBf7fRchAmICegpST4XQEAImE+/PPeGgBDQU5J8LoCAEDCffnnuDQFfKGA8HqerqyvPBr9fUFtbS93d3e6Xfh90fG8ueCEEfKGAYZhDwA/dg4AQMMzPUuhrIWAAAWOxGDU2NoZu+snJCc3MzDy5D/4iOnR7ZW8Q9BfMua4WAua6o8L2g4CFBYYjOMARXAg0mICF6DLje2ACFhYOJiAmYGGNy7gbBISAELCoHci4edB3OHKdPfUdk1zvzXE/TECOVAxlgoCGYHMsFQJypGIoEwQ0BJtjqRCQIxVDmSCgIdgcS4WAHKkYygQBDcHmWCoE5EjFUCYIaAg2x1IhIEcqhjJBQEOwOZYKATlSMZQJAhqCzbFUCMiRiqFMENAQbI6lQkCOVAxlgoCGYHMsFQJypGIoEwQ0BJtjqRCQIxVDmSCgIdgcS4WAHKkYygQBDcHmWCoE5EjFUCYIaAg2x1IhIEcqhjJBQEOwOZYKATlSMZQJAhqCzbFUCMiRiqFMENAQbI6lQkCOVAxlgoCGYHMsFQJypGIoEwQ0BJtjqRCQIxVDmf4DPlsr1QN1W8wAAAAASUVORK5CYII=</thumbnail><stage name="Stage" guid="28495c21-e933-460a-a91e-dd9348770d8c" width="480" height="360" costume="0" tempo="60" threadsafe="false" lines="round" codify="false" inheritance="false" sublistIDs="false" scheduled="false" id="1"><pentrails>data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAeAAAAFoCAYAAACPNyggAAAU30lEQVR4Xu3dzW0mxxEG4CGjsCNQANLVPwEsFtyzfFcYUhh2APaZIBiAYF8VgQKQglgsaUCSjbW1FGo470z19Dw+11fV/XQJLygDo5vF/wgQIECAAIHDBW4On2ggAQIECBAgsAhgS0CAAAECBBoEBHADupEECBAgQEAA2wECBAgQINAgIIAb0I0kQIAAAQIC2A4QIECAAIEGAQHcgG4kAQIECBAQwHaAAAECBAg0CAjgBnQjCRAgQICAALYDBAgQIECgQUAAN6AbSYAAAQIEBLAdIECAAAECDQICuAHdSAIECBAgIIDtAAECBAgQaBAQwA3oRhIgQIAAAQFsBwgQIECAQIOAAG5AN5IAAQIECAhgO0CAAAECBBoEBHADupEECBAgQEAA2wECBAgQINAgIIAb0I0kQIAAAQIC2A4QIECAAIEGAQHcgG4kAQIECBAQwHaAAAECBAg0CAjgBnQjCRAgQICAALYDBAgQIECgQUAAN6AbSYAAAQIEBLAdIECAAAECDQICuAHdSAIECBAgIIDtAAECBAgQaBAQwA3oRhIgQIAAAQFsBwgQIECAQIOAAG5AN5IAAQIECAhgO0CAAAECBBoEBHADupEECBAgQEAA2wECBAgQINAgIIAb0I0kQIAAAQIC2A4QIECAAIEGAQHcgG4kAQIECBAQwHaAAAECBAg0CAjgBnQjCRAgQICAALYDBAgQIECgQUAAN6AbSYAAAQIEBLAdIECAAAECDQICuAHdSAIECBAgIIDtAAECBAgQaBAQwA3oRhIgQIAAAQFsBwgQIECAQIOAAG5AN5IAAQIECAhgO0CAAAECBBoEBHADupEECBAgQEAA2wECBAgQINAgIIAb0I0kQIAAAQIC2A4QIECAAIEGAQHcgG4kAQIECBAQwHaAAAECBAg0CAjgBnQjCRAgQICAALYDBAgQIECgQUAAN6AbSYAAAQIEBLAdIECAAAECDQICuAHdSAIECBAgIIDtAAECBAgQaBAQwA3oRhIgQIAAAQFsBwgQIECAQIOAAG5AN5IAAQIECAhgO0CAAAECBBoEBHADupEECBAgQEAA2wECBAgQINAgIIAb0I0kQIAAAQIC2A4QIECAAIEGAQHcgG4kAQIECBAQwHaAAAECBAg0CAjgBnQjCRAgQICAALYDBAgQIECgQUAAN6AbSYAAAQIEBLAdIECAAAECDQICuAHdSAIECBAgIIDtAAECBAgQaBAQwA3oRhIgQIAAAQFsBwgQIECAQIOAAG5AN5IAAQIECAhgO0CAAAECBBoEBHADupEECBAgQEAA2wECBAgQINAgIIAb0I0kQIAAAQIC2A4QIECAAIEGAQHcgG4kAQIECBAQwHaAAAECBAg0CAjgBnQjCRAgQICAALYDBAgQIECgQUAAN6AbSYAAAQIEBLAdIECAAAECDQICuAHdSAIECBAgIIDtAAECBAgQaBAQwA3oRhIgQIAAAQFsBwgQIECAQIOAAG5AN5IAAQIECAhgO0CAAAECBBoEBHADupEECBAgQEAA2wECBAgQINAgIIAb0I0kQIAAAQIC2A4QIECAAIEGAQHcgG4kAQIECBAQwHaAAAECBAg0CAjgBnQjCRAgQICAALYDBAgQIECgQUAAN6AbSYAAAQIEBLAdIECAAAECDQICuAHdSAIECBAgIIDtAAECBAgQaBAQwA3oRhIgQIAAAQFsBwgQIECAQIOAAG5AN5IAAQIECAhgO0CAAAECBBoEBHADupEECBAgQEAA2wECBAgQINAgIIAb0I0kQIAAAQIC2A4QIECAAIEGAQHcgG4kAQIECBAQwHaAAAECBAg0CAjgBnQjCRAgQICAALYDBAgQIECgQUAAN6AbSYAAAQIEBLAdIECAAAECDQICuAHdSAIECBAgIIDtAAECBAgQaBAQwA3oRhIgQIAAAQFsBwgQIECAQIOAAG5AN5IAAQIECAhgO0CAAAECBBoEBHADupEECBAgQEAA2wECBAgQINAgIIAb0I0kQIAAAQIC2A4QIECAAIEGAQHcgG4kAQIECBAQwHaAAAECBAg0CAjgBnQjCRAgQICAALYDBAgQIECgQUAAN6AbSYAAAQIEBLAdIECAAAECDQICuAHdSAIECBAgIIDtAAECBAgQaBAQwA3oRhIgQIAAAQFsBwgQIECAQIOAAG5AN5IAAQIECAhgO0BgpcDd3d3nT09PNw8PD9+t/KlyAgQI/FdAAFsGAisF7u7uvl6W5eb+/v6blT9VToAAAQFsBwi8VkAAv1bO7wgQ+FjAX8D2gcBKAQG8Ekw5AQKfFBDAFoPASgEBvBJMOQECAtgOEEgICOCEoh4ECPgL2A4QWCkggFeCKSdAwF/AdoBAQkAAJxT1IEDAX8B2gMBKAQG8Ekw5AQL+ArYDBBICAjihqAcBAv4CtgMEVgpUA/ju7u4vy7L8YVmWH1aOUP6CwM3Nze/fv3//r8fHx79DInB2AQF89hd0/sMFVgTw35Zl+WxZlm8PP+S8A//09PT0/cPDw1fzXtHNriIggK/y0u4ZE1gRwD5ZGVP/uVHVPjxWOwK7CAjgXVg1nVmgGgLVupmt0ndjmhbVr1NAAHfqm31KgWoIVOtOidB0aKZN8MbuIiCAd2HVdGaBaghU62a2St+NaVpUv04BAdypb/YpBaohUK07JULToZk2wRu7i4AA3oVV05kFqiFQrZvZKn03pmlR/ToFBHCnvtmnFKiGQLXulAhNh2baBG/sLgICeBdWTWcWqIZAtW5mq/TdmKZF9esUEMCd+mafUqAaAivqLv/FrOoXrqqmp1wsh76cgAC+3JO78FaBagisqPPFrGUpfeGqarr1jf2ewBECAvgIZTOmEqiGQLpuKsT/uwyrmV/X3V4SEMB2g8BKgXRYVPutPOapyqsG1bpTXd5hLysggC/79C7+WoFqCKTrXnveM/yO1RleyRnTAgI4Larf9ALpsKj2mxm2alCtm9nK3eYREMDzvKWbHCRQDYF03UHXaxnDqoXd0GYBAdz8AMafTyAdFtV+55Oqn7hqUK2rT1ZJoE9AAPfZm3xSgWoIpOtOylU6NqsSk6LJBATwZA/qOvsLpMOi2m//m/VNqBpU6/puYjKBuoAArlupJPCTQDUEdqg73Rez0l+4qppaVQJnEBDAZ3glZxxKoBoCO9Sd8YtZ0S9cVU2HWhiHIfCCgAC2GgRWClRDoKtu5XV2LWewK6/mJxcQwCd/QMc/XqArVKpzjxd5eWL1zOm6kQychcBLAgLYbhBYKZAOi3S/ldfZtTx9t2q/XS+lOYGQgAAOQWpzHYFqCHTVjfQSDEZ6DWcZTUAAj/YizjO8QFeoVOeOBFg9c7puJANnIeBfQdsBAiGBdFik+4WuGWmTvlu1X+TwmhDYWcBfwDsDaz+fQDUEuupGEmcw0ms4y2gCAni0F3Ge4QW6QqU6dyTA6pnTdSMZOAsB/wraDhAICaTDYod+u38xq+sLV1Wr0FNrQ2BXAX8B78qr+YwC1RBorDvii1ktX7iqms64d+40n4AAnu9N3WhngWoIjF63hWnmu21x8VsCawQE8BottQR6/2MMXy/LcnN/f//Nbz1ENRy3PGZ1Rlfdlrv5LYGjBATwUdLmTCPQFSrpuVseJH2WdL8td/NbAkcJCOCjpM2ZRiAdFl39tjxI15mrc7fczW8JHCUggI+SNmcagWoIjF635UFmvtsWF78lsEZAAK/RUkvA/wf80w4IYP8oENguIIC3G+pwMYHRw6d6vi3PVp3RVbflbn5L4CgBAXyUtDnTCHSFSnrulgdJnyXdb8vd/JbAUQIC+Chpc6YRSIdFY7/Pl2V58/HDjP6Fq6rVNMvmIlMLCOCpn9fl9hCohsDodW/fvv3i9vb2fwJ4WZahv3BVNd3j3fUkkBYQwGlR/aYXqIbA6HWfeqjRz1w93/RL6IJTCAjgKZ7RJY4UqIbA6HUC+MitMYvArwUEsK0gsFJg9GCtnk8Ar3x45QTCAgI4DKrd/ALVgBu9TgDPv6tuOLaAAB77fZxuQIHRg7V6PgE84HI50qUEBPClnttlEwLVgBu9TgAntkEPAq8XEMCvt/PLiwqMHqzV8wngiy6waw8jIICHeQoHOYtANeBGrxPAZ9k455xVQADP+rLutZvA6MH67t27vz4/P3+2LMu3r0D489PT0/cPDw9f/dZvRzd4xb39hMDhAgL4cHIDzy4wevi8efPmy9vb2z/e3t7+uNb6+fn5dx8+fPjn4+PjPwTwWj31BNYJCOB1XqoJ+E/x+c8R+qeAQERAAEcYNbmSwOh/AR/xFgyOUDZjdgEBPPsLu19cQPgs/i1AfKs0vKKAAL7iq7vzJgEBLIA3LZAfE/hFQABbBQIrBQSwAF65MsoJfFJAAFsMAisFBLAAXrkyygkIYDtAICEggAVwYo/0IOAvYDtAYKWAABbAK1dGOQF/AdsBAgmBrgDe+IWrxNU/7jH0F7PSl9WPwB4C/gLeQ1XPqQW6AnjLF67SDzL6F7PS99WPwB4CAngPVT2nFugK4DOisjrjqznzUQIC+Chpc6YRECr1p2RVt1J5PQEBfL03d+ONAkKlDsiqbqXyegIC+Hpv7sYbBYRKHZBV3Url9QQE8PXe3I03CgiVOiCrupXK6wkI4Ou9uRtvFBAqdUBWdSuV1xMQwNd7czfeKCBU6oCs6lYqrycggK/35m68UUCo1AFZ1a1UXk9AAF/vzd14o0A6VAb7wtVGnV/9vOWLWelL6EdgDwEBvIeqnlMLpAN4pC9cpR+u64tZ6XvoR2APAQG8h6qeUwukA3hqrOLlmBahlE0lIICnek6XOUJAWOSVmeZNdRxfQACP/0ZOOJiAsMg/CNO8qY7jCwjg8d/ICQcTEBb5B2GaN9VxfAEBPP4bOeFgAsIi/yBM86Y6ji8ggMd/IyccTEBY5B+Ead5Ux/EFBPD4b+SEgwkIi/yDMM2b6ji+gAAe/42ccDABYZF/EKZ5Ux3HFxDA47+REw4mUA2Lyb9wlX6V6Bez0ofTj8AeAgJ4D1U9pxaoBvDMX7hKP3D6i1np8+lHYA8BAbyHqp5TC1QDeGqEpsuxb4I3dhcBAbwLq6YzCwiBvtdl32dvcl5AAOdNdZxcQAj0PTD7PnuT8wICOG+q4+QCQqDvgdn32ZucFxDAeVMdJxcQAn0PzL7P3uS8gADOm+o4uYAQ6Htg9n32JucFBHDeVMfJBYRA3wOz77M3OS8ggPOmOk4uIAT6Hph9n73JeQEBnDfVcXIBX7hqfeDSF7NaT2g4gaKAAC5CKSPwHwFfuOrbheoXs/pOaDKBuoAArlupJECAAAECMQEBHKPUiAABAgQI1AUEcN1KJQECBAgQiAkI4BilRgQIECBAoC4ggOtWKgkQIECAQExAAMcoNSJAgAABAnUBAVy3UkmAAAECBGICAjhGqREBAgQIEKgLCOC6lUoCBAgQIBATEMAxSo0IECBAgEBdQADXrVQSIECAAIGYgACOUWpEgAABAgTqAgK4bqWSAAECBAjEBARwjFIjAgQIECBQFxDAdSuVBAgQIEAgJiCAY5QaESBAgACBuoAArlupJECAAAECMQEBHKPUiAABAgQI1AUEcN1KJQECBAgQiAkI4BilRgQIECBAoC4ggOtWKgkQIECAQExAAMcoNSJAgAABAnUBAVy3UkmAAAECBGICAjhGqREBAgQIEKgLCOC6lUoCBAgQIBATEMAxSo0IECBAgEBdQADXrVQSIECAAIGYgACOUWpEgAABAgTqAgK4bqWSAAECBAjEBARwjFIjAgQIECBQFxDAdSuVBAgQIEAgJiCAY5QaESBAgACBuoAArlupJECAAAECMQEBHKPUiAABAgQI1AUEcN1KJQECBAgQiAkI4BilRgQIECBAoC4ggOtWKgkQIECAQExAAMcoNSJAgAABAnUBAVy3UkmAAAECBGICAjhGqREBAgQIEKgLCOC6lUoCBAgQIBATEMAxSo0IECBAgEBdQADXrVQSIECAAIGYgACOUWpEgAABAgTqAgK4bqWSAAECBAjEBARwjFIjAgQIECBQFxDAdSuVBAgQIEAgJiCAY5QaESBAgACBuoAArlupJECAAAECMQEBHKPUiAABAgQI1AUEcN1KJQECBAgQiAkI4BilRgQIECBAoC4ggOtWKgkQIECAQExAAMcoNSJAgAABAnUBAVy3UkmAAAECBGICAjhGqREBAgQIEKgLCOC6lUoCBAgQIBATEMAxSo0IECBAgEBdQADXrVQSIECAAIGYgACOUWpEgAABAgTqAgK4bqWSAAECBAjEBARwjFIjAgQIECBQFxDAdSuVBAgQIEAgJiCAY5QaESBAgACBuoAArlupJECAAAECMQEBHKPUiAABAgQI1AUEcN1KJQECBAgQiAkI4BilRgQIECBAoC4ggOtWKgkQIECAQExAAMcoNSJAgAABAnUBAVy3UkmAAAECBGICAjhGqREBAgQIEKgLCOC6lUoCBAgQIBATEMAxSo0IECBAgEBdQADXrVQSIECAAIGYgACOUWpEgAABAgTqAgK4bqWSAAECBAjEBARwjFIjAgQIECBQFxDAdSuVBAgQIEAgJiCAY5QaESBAgACBuoAArlupJECAAAECMQEBHKPUiAABAgQI1AUEcN1KJQECBAgQiAkI4BilRgQIECBAoC4ggOtWKgkQIECAQExAAMcoNSJAgAABAnUBAVy3UkmAAAECBGICAjhGqREBAgQIEKgLCOC6lUoCBAgQIBATEMAxSo0IECBAgEBdQADXrVQSIECAAIGYgACOUWpEgAABAgTqAgK4bqWSAAECBAjEBARwjFIjAgQIECBQFxDAdSuVBAgQIEAgJiCAY5QaESBAgACBuoAArlupJECAAAECMQEBHKPUiAABAgQI1AUEcN1KJQECBAgQiAkI4BilRgQIECBAoC4ggOtWKgkQIECAQExAAMcoNSJAgAABAnUBAVy3UkmAAAECBGICAjhGqREBAgQIEKgLCOC6lUoCBAgQIBATEMAxSo0IECBAgEBdQADXrVQSIECAAIGYgACOUWpEgAABAgTqAgK4bqWSAAECBAjEBARwjFIjAgQIECBQFxDAdSuVBAgQIEAgJiCAY5QaESBAgACBusC/AQfxJPCZsPsLAAAAAElFTkSuQmCC</pentrails><costumes><list id="2"></list></costumes><sounds><list id="3"></list></sounds><variables></variables><blocks></blocks><scripts></scripts><sprites><watcher var="length" style="normal" x="10" y="10" color="243,118,29"/><sprite name="Sprite" idx="1" x="-34" y="33" heading="90" scale="1" rotation="1" draggable="true" costume="0" color="80,80,80" pen="tip" id="9"><costumes><list id="10"></list></costumes><sounds><list id="11"></list></sounds><variables></variables><blocks></blocks><scripts><script x="86" y="41"><block id="4687" s="receiveGo"></block><custom-block id="4833" s="squiral %s"><l>5</l></custom-block></script></scripts></sprite></sprites></stage><hidden></hidden><headers></headers><code></code><blocks><block-definition s="squiral %&apos;sides&apos;" type="command" category="pen" guid="a00b8903-07cf-4265-ad03-a66a1692cb34" isImported="false"><header></header><code></code><inputs><input type="%s"></input></inputs><script><block id="4752" s="doDeclareVariables"><list><l>length</l></list></block><block id="4754" s="doSetVar"><l>length</l><l>5</l></block><block id="4904" s="down"></block><block id="4756" s="doRepeat"><block id="4759" s="reportProduct"><l>4</l><block id="4758" var="sides"/></block><script><block id="4776" s="forward"><block id="4778" var="length"/></block><block id="4777" s="turn"><l>90</l></block><block id="4779" s="doChangeVar"><l>length</l><l>5</l></block></script></block><block id="4945" s="up"></block></script></block-definition></blocks><variables><variable name="length"><l>0</l></variable></variables><editing><block-definition s="squiral %&apos;sides&apos;" type="command" category="pen" guid="a00b8903-07cf-4265-ad03-a66a1692cb34" isImported="false"><header></header><code></code><inputs><input type="%s"></input></inputs><script><block id="4752" s="doDeclareVariables"><list><l>length</l></list></block><block id="4754" s="doSetVar"><l>length</l><l>5</l></block><block id="4904" s="down"></block><block id="4756" s="doRepeat"><block id="4759" s="reportProduct"><l>4</l><block id="4758" var="sides"/></block><script><block id="4776" s="forward"><block id="4778" var="length"/></block><block id="4777" s="turn"><l>90</l></block><block id="4779" s="doChangeVar"><l>length</l><l>5</l></block></script></block><block id="4945" s="up"></block></script></block-definition></editing></project>