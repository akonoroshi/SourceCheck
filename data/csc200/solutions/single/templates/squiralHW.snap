snapshot {
    stage:Stage {
        sprite:Sprite {
            script {
                receiveGo
                evaluateCustomBlock(literal)
            }
        }
    }
    customBlock {
        script {
            doDeclareVariables(list(varMenu:length))
            doSetVar(varMenu:length, literal)
            down
            doRepeat(reportProduct(literal:4, var:sides), script {
                forward(var:length)
                turn(literal:90)
                doChangeVar(varMenu:length, literal)
            })
            up
        }
        varDec:sides
    }
    varDec:length
}